### ISSUE TITLE
Would you please publish the JAR that comes with this R package as a Spark Package

### ISSUE THREAD

**Issue by falaki** [rstudio/sparklyr#171]
It will make installation of the R package easier in other Spark environments especially with regards to class path management. 

Spark Packages website: https://spark-packages.org/
Instructions on how to register a package: https://spark-packages.org/register/explain

**Comment by jjallaire** [comment]
We'll consider doing this when we publish a stable version of the package
to CRAN (should be in the next 2 or 3 weeks)

**Comment by falaki** [comment]
Is there still interest in this?

**Comment by jjallaire** [comment]
Yes, getting the deployment topology right in all environments is our top
focus right now. Interestingly, we've heard from other
customers/environments that any use of Spark packages is problematic, see:
https://github.com/rstudio/sparklyr/pull/314

I think we'll need to do both and allow users to configure the behavior via
a global option.

@javierluraschi
<https://github.com/rstudio/rstudio/commits/master?author=javierluraschi>
@kevinushey

**Comment by javierluraschi** [comment]
Just submitted `sparklyr` for verification to https://spark-packages.org/ (which seems to be the first step), I'll keep this thread posted. The plan would be to submit the CRAN version `0.4` and update them both (CRAN and https://spark-packages.org/) as new releases of `sparklyr` become available. @jjallaire
